arch/cpu/native/net/tun6-net.c

```diff
@@ -95,35 +104,144 @@ ssystem(const char *fmt, ...)
   return system(cmd);
 }
 
+/*---------------------------------------------------------------------------*/
+static bool
+set_default_prefix(const char *prefix)
+{
+  char *ipaddr = strdup(prefix);
+  if(!ipaddr) {
+    return false;
+  }
+
+  uip_ip6addr_t prefix_addr;
```

Comment:
I think this line can be pushed inside the if-statement a few lines below.

Reply:
Fixed.